workers/unity/Packages/com.improbable.gdk.transformsynchronization/Systems/RateLimitedTransformSendSystem.cs

```diff
@@ -51,12 +51,23 @@ protected override void OnUpdate()
                     Rotation = transformToSend.Orientation.ToImprobableQuaternion(),
                     Velocity = transformToSend.Velocity.ToImprobableVelocity(),
                     PhysicsTick = transform.PhysicsTick + data.TicksSinceLastUpdate[i].NumberOfTicks,
-                    TicksPerSecond = tickRate.PhysicsTicksPerRealSecond,
-                    DirtyBit = true
+                    TicksPerSecond = tickRate.PhysicsTicksPerRealSecond
                 };
 
-                if (!(TransformUtils.HasChanged(currentTransform.Location, transform.Location) ||
-                    TransformUtils.HasChanged(currentTransform.Rotation, transform.Rotation)))
+                var locationHasChanged = TransformUtils.HasChanged(currentTransform.Location, transform.Location);
+                var rotationHasChanged = TransformUtils.HasChanged(currentTransform.Rotation, transform.Rotation);
+                var velocityHasChanged = TransformUtils.HasChanged(currentTransform.Velocity, transform.Velocity);
```

Comment:
This is deliberately not changing velocity right now.

Reply:
Reverted to the old logic.